Automattic/wp-calypso
# Updated sidebar to load OPENED in Reader. (#64632)

#### Proposed Changes

* Updated sidebar to load OPENED always in Reader
* wrapped Sidebar component into a funtional react, which updates the sidebar collapsed state before loading of Sidebar component.

#### Testing Instructions

Steps to reproduce / check the issue:
1. Go to "My Sites" section.
2. Collapse the Sidebar.
3. Now go to "Reader" section.
4. Notice that sidebar remains collapsed & malfunctions as shown in below 'Before' screenshots.


Before:

Reader section sidebar:

![image](https://user-images.githubusercontent.com/3984908/178989337-4fc5842f-e8a2-4f37-a6ff-7b79a5e10ba2.png)

Icons not making any sense: 

![image](https://user-images.githubusercontent.com/3984908/179055454-9c0586f9-458e-4946-a9c9-572e1869e71f.png)



After:
<img width="871" alt="Screenshot 2022-07-14 at 11 54 35 PM" src="https://user-images.githubusercontent.com/3984908/179055316-c3f278c7-37e2-41f3-83ef-d4270ecf0519.png">

The Sidebar is always opened in Reader.



*

fixes: https://github.com/Automattic/wp-calypso/issues/64632



Related to #64632
